src/Common/src/System/Net/Logging/GlobalLog.cs

```diff
@@ -173,41 +183,52 @@ public static void Enter(string functionName, string parameters)
             EventSourceLogging.Log.FunctionStart(functionName, parameters);
         }
 
-        public static void Assert(bool condition, string messageFormat, params object[] data)
+        public static void Assert(bool condition, string message)
```

Comment:
It turns out there were only ~10 call sites remaining, so I've fixed those up in this project as well.